cmake/std/atdm/ats2/trilinos_jsrun

```diff
@@ -172,9 +172,6 @@ if [[ "$TPETRA_ASSUME_CUDA_AWARE_MPI" == "1" ]] && [[ "$np_is_one" == "false" ]]
     # prepend the argument
     args=("-M -gpu" "${args[@]}")
   fi
-
-  # Point jsrun to pami
-  args=("-E LD_PRELOAD=$OPAL_PREFIX/lib/pami_451/libpami.so" "${args[@]}")
```

Comment:
@jjellio, as we discussed at our meeting today, can you please wrap this statement in:

```
if [[ "${TRILINOS_JSRUN_SET_LD_REPLOAD}" == "1" ]]; then
  args=(...)
fi
```

?

Then we will leave this env var `TRILINOS_JSRUN_SET_LD_REPLOAD` unset by default but we will set it to `1` in the file:

* [Trilinos/cmake/ctest/drivers/atdm/ats2/local-driver.sh](https://github.com/trilinos/Trilinos/blob/develop/cmake/ctest/drivers/atdm/ats2/local-driver.sh)

so the automated tests get this and we will tell the user to set it it when running `ctest` locally in the ATS-2 section in:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/std/atdm/README.md#ats-2